### PR TITLE
Header

### PR DESCRIPTION
**Pull request by alex-ko1**
Changed header and register page, added sheet page.

### CODE REVIEW

File: src/views/SignUpPage.vue [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+<!--                            <input type="password" name="password" id="password" placeholder="••••••••" class="bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" required>-->
+<!--                        </div>-->
+<!--                        <div class="flex items-center justify-between">-->
+<!--                            <div class="flex items-start">-->
+<!--                                <div class="flex items-center h-5">-->
+<!--                                    <input id="remember" aria-describedby="remember" type="checkbox" class="w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-primary-300 dark:bg-gray-700 dark:border-gray-600 dark:focus:ring-primary-600 dark:ring-offset-gray-800">-->
+                        <!---->
+
+                        <oe-input v-for="(inp,i) in inputInfo" :key="i" :inputInfo="inputInfo[i]" @onInput="onInput(i, $event)"/>
```
Comment (matrunchyk) [inline src/views/SignUpPage.vue:25]: Wanna remove this?

File: src/components/AppHeader.vue [hunk 2]
```diff
@@ -145,8 +82,8 @@
                                 Cloud</a>
                         </li>
                         <li>
-                            <a href="#"
-                               class="block py-2 pr-4 pl-3 text-primary-600 text-gray-700 border-b border-gray-100 hover:bg-gray-50 lg:hover:bg-transparent lg:border-0 lg:hover:text-primary-700 lg:p-0 dark:text-gray-400 lg:dark:hover:text-white dark:hover:bg-gray-700 dark:hover:text-white lg:dark:hover:bg-transparent dark:border-gray-700" aria-current="page">Sheets</a>
+                            <RouterLink to="/sheet-editor"
```
Comment (matrunchyk) [inline src/components/AppHeader.vue:85]: Shouldn't it be /sheet?

**Comment by matrunchyk** [comment]
screenshot

Can you please also update it here?


<details><summary>Browser metadata</summary>





```
Path:      /forgot-password
Browser:   Chrome 112.0.0.0 on Mac OS 10.15.7
Viewport:  1728 x 968 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-11--enchanting-gingersnap-4c2959.netlify.app%2Fforgot-password&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=112.0&resolution=1728x968&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-11--enchanting-gingersnap-4c2959.netlify.app/forgot-password) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=64500466227d0c00086f4eea&commentID=6450094d32a3bd164c97466f&resolution=resolved)

File: src/components/AppHeader.vue (hunk 2 above)
Comment (alex-ko1) [reply src/components/AppHeader.vue]: Fixed

**Comment by matrunchyk** [comment]
screenshot

Something wrong with these menu items. Please check how it looks here https://openeditor.cloud/home-page
<details><summary>Browser metadata</summary>





```
Path:      /home-page
Browser:   Chrome 112.0.0.0 on Mac OS 10.15.7
Viewport:  1728 x 968 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-11--enchanting-gingersnap-4c2959.netlify.app%2Fhome-page&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=112.0&resolution=1728x968&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-11--enchanting-gingersnap-4c2959.netlify.app/home-page) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=64500b173795720008589046&commentID=64500c2432a3bd196a974567&resolution=resolved)

File: src/views/SignUpPage.vue [hunk 3]
```diff
@@ -1,49 +1,94 @@
...
-<!--                                    <input id="remember" aria-describedby="remember" type="checkbox" class="w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-primary-300 dark:bg-gray-700 dark:border-gray-600 dark:focus:ring-primary-600 dark:ring-offset-gray-800">-->
-                        <!---->
-
-                        <oe-input v-for="(inp,i) in inputInfo" :key="i" :inputInfo="inputInfo[i]" @onInput="onInput(i, $event)"/>
-                        <div class="flex items-center justify-between">
-                            <div class="flex items-start">
-                                <div class="flex items-center h-5">
-                                    <input id="remember" aria-describedby="remember" type="checkbox" class="w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-primary-300 dark:bg-gray-700 dark:border-gray-600 dark:focus:ring-primary-600 dark:ring-offset-gray-800">
-
+                        <oe-input v-for="(inp,i) in inputInfo" :key="i" :inputInfo="inputInfo[i]" @onInput="onInput(i, $event)" class="block"/>
```
Comment (matrunchyk) [inline src/views/SignUpPage.vue:16]: app-input?

Comment (alex-ko1) [reply src/views/SignUpPage.vue]: fixed

**Comment by matrunchyk** [comment]
screenshot

I think we don't need both. In fact,  "Log out" should be removed, cause we have "Sign out" inside the menu.

<details><summary>Browser metadata</summary>





```
Path:      /home-page
Browser:   Chrome 112.0.0.0 on Mac OS 10.15.7
Viewport:  1728 x 968 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-11--enchanting-gingersnap-4c2959.netlify.app%2Fhome-page&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=112.0&resolution=1728x968&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-11--enchanting-gingersnap-4c2959.netlify.app/home-page) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6450f7903a73c100079fda7d&commentID=6450fd9d97ca7f1012553c93&resolution=resolved)

**Review by dddarkasss — APPROVED** [review]

**Pull request merged**